src/shared/components/person/settings.tsx

```diff
@@ -332,8 +340,18 @@ export class Settings extends Component<any, SettingsState> {
   }
 
   render() {
+    /* eslint-disable jsx-a11y/anchor-has-content, jsx-a11y/anchor-is-valid */
     return (
       <div className="person-settings container-lg">
+        <a
+          ref={this.exportSettingsLink}
+          download={`lemmy-settings-${UserService.Instance.myUserInfo?.local_user_view.person.name}.json`}
```

Comment:
Please suggest a better default file name if you have one.

Reply:
Looking at another backup program, I'd say do something like:

`lemmy_user_settings.${(new Date()).toISOString().replace('/:/g','').replace('/-/g','')}.json`

So it'll look like `lemmy_user_settings.20231119T000Z.json`

Removing colons and dashes are good to make iso8601 date strings file-system safe everywhere. Might be good to extract that to its own function too.